gdnative-core/src/core_types/quat.rs

```diff
@@ -0,0 +1,46 @@
+use crate::core_types::Real;
+use glam::Quat as GQuat;
```

Comment:
Not sure what the godot-rust style is here, but it could also simply be an explicit `glam::Quat` qualification on usage.